website/docs/r/vapp_vm.html.markdown

```diff
@@ -442,7 +442,36 @@ resource "vcd_vapp_vm" "advancedVM" {
   cpu_reservation = "200"
   cpu_limit       = "1000"
 }
+```
+
+## Example Usage (VM copy)
+This example shows how to create a VM copy from existing VM
+
+```hcl
+
+data "vcd_vapp_vm" "existing" {
+  vapp_name = data.vcd_vapp.web.name
+  name      = "web1"
+}
+
+resource "vcd_vapp_vm" "vm-copy" {
+  org = "org"
+  vdc = "vdc"
 
+  copy_from_vm_id = data.vcd_vapp_vm.existing.id # source VM ID
+  vapp_name       = vcd_vapp.vm-copy-destination-template-vm.name
+  name            = "VM Copy"
+  power_on        = false
+
+  network {
+    type               = "org"
+    name               = vcd_vapp_org_network.template-vapp-copy.org_network_name
```

Comment:
Where does this vApp network come from?

Reply:
I obviously didn't have a data source for it, but changed now - added additional data source.